scripts/deploy_006_migration.py

```diff
@@ -0,0 +1,176 @@
+import os
+
+from brownie import *
+from brownie.network import priority_fee
+from common import *
+
+VEOGV_PROXY_ADDRESS = "0x0c4576ca1c365868e162554af8e385dc3e7c66d9"
+OGV_PROXY_ADDRESS = "0x9c354503c38481a7a7a51629142963f98ecc12d0"
+REWARDS_PROXY_ADDRESS = "0x7d82e86cf1496f9485a8ea04012afeb3c7489397"
+
+OGN_ADDRESS = "0x8207c1ffc5b6804f6024322ccf34f29c3541ae26"
+
+OUSD_BUYBACK_PROXY_ADDRESS = "0xD7B28d06365b85933c64E11e639EA0d3bC0e3BaB"
+OETH_BUYBACK_PROXY_ADDRESS = "0xFD6c58850caCF9cCF6e8Aee479BFb4Df14a362D2"
+
+# TODO: Update after deploying buyback contracts
+OUSD_BUYBACK_NEW_IMPL_ADDRESS = "0xdd753cb1fc4e8c72c5a40254ac064515301d11e0"
+OETH_BUYBACK_NEW_IMPL_ADDRESS = "0x6c952cc410a253e70f3383028c7e0a7f477b921c"
+
+def main():
```

Comment:
It would be good to have some sanity checking fork testing on this - these are fairly major changes. 

I think we should also mint the new OGN to the migrator during this governance action. We can defer team funds for separate actions. But we still should fork test this migration +  team funds + activating the migration and doing a migration.

Reply:
Have added a script that runs at the end of the deploy `006`